client/src/app/site/agenda/components/agenda-list/agenda-list.component.ts

```diff
@@ -2,7 +2,6 @@ import { Component, OnInit } from '@angular/core';
 import { Router, ActivatedRoute } from '@angular/router';
 import { Title } from '@angular/platform-browser';
 import { MatSnackBar } from '@angular/material';
-
 import { TranslateService } from '@ngx-translate/core';
```

Comment:
the empty line was meant to be.
https://angular.io/guide/styleguide#import-line-spacing

Reply:
We really need some common understanding on which guidelines to follow and which not. The document you linked also claims it to be a good idea to sort alphabetically, which as of now is seldom the case anywhere.